src/python/pants/backend/python/dependency_inference/parse_python_dependencies.py

```diff
@@ -31,21 +34,44 @@ class ParsedPythonImports(FrozenDict[str, ParsedPythonImportInfo]):
     """All the discovered imports from a Python source file mapped to the relevant info."""
 
 
+class ParsedPythonAssets(DeduplicatedCollection[Tuple[str, str]]):
+    """All the discovered possible assets from a Python source file.
+
+    The tuple is of (containing module, filename).
```

Comment:
Ah I didn't grok this well enough and it resulted in some of my confusion. I think an example could help, e.g. `("path.to.module", "foo.txt")`.